Formula/gnupg.rb

```diff
@@ -49,11 +50,22 @@ def install
     system "make"
     system "make", "check"
     system "make", "install"
+
+    # Configure scdaemon as recommended by upstream developers
+    # https://dev.gnupg.org/T5415#145864
+    on_macos do
+      # write to buildpath then install to ensure existing files are not clobbered
+      (buildpath/"scdaemon.conf").write <<~EOS
+        disable-ccid
+      EOS
+      pkgetc.install "scdaemon.conf"
+    end
   end
 
   def post_install
     (var/"run").mkpath
     quiet_system "killall", "gpg-agent"
+    quiet_system "killall", "scdaemon"
```

Comment:
```suggestion
    quiet_system "gpgconf", "--reload", "all"
```
Apologies for the delayed review on this bit, but I think this change would be an improvement on what's here. We may as well do this now too while we're making changes here. (Please squash this with your existing commit.)

Reply:
applied!